File: server/src/main/java/com/java/otus/architect/page/Page.java
package com.java.otus.architect.page;

import java.util.List;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class Page {

  private String email;
  private String firstName;
  private String lastName;
  private String city;
  private Gender gender;
  private int age;
  private List<String> areasOfInterest;

}

File: front/src/models/Page.ts
export enum Gender {
  MALE = 'MALE',
  FEMALE = 'FEMALE'
}

export class Page {
  constructor(
    public email: string,
    public firstName: string,
    public lastName: string,
    public city: string,
    public gender: Gender,
    public age: number,
    public areasOfInterest: string[]
  ) {}
}
